core/layout_api.php

```diff
@@ -47,16 +47,20 @@
 
 /**
  * Print the page header section
- * @param string $p_page_title   Html page title.
- * @param string $p_redirect_url URL to redirect to if necessary.
- * @param string $p_page_id      The page id.
+ * @param string $p_page_title    Html page title.
+ * @param string $p_redirect_url  URL to redirect to if necessary.
+ * @param string $p_page_id       The page id.
+ * @param string $p_canonical_url Canonical URL if necessary.
```

Comment:
Probably worth repeating that a relative URL is expected here (same as PHPDoc of new html_meta_canonical() function).

Reply:
Ok. Made change for `$p_redirect_url` as well for consistency.